### ISSUE TITLE
nestRemoting does not appear

### ISSUE THREAD

**Issue by Lughino** [strongloop/loopback-component-explorer#167]
I added a layer to the rest api but this does not appear in the component explorer.

``` javascript
// member.json
"relations": {
    "corporates": {
      "type": "hasMany",
      "model": "Corporate",
      "foreignKey": "",
      "options": {
        "validate": true,
        "forceId": false
      }
    },
   ...
}

// corporate.json
"relations": {
    "contests": {
      "type": "hasMany",
      "model": "Contest",
      "foreignKey": "",
      "options": {
        "validate": true,
        "forceId": false
      }
    },
    "member": {
      "type": "belongsTo",
      "model": "Member",
      "foreignKey": "",
      "options": {
        "validate": true,
        "forceId": false
      }
    }
  },

// in boot script
Member.nestRemoting('corporates');
```

Now I can make calls, eg with postman, like this:

```
/api/Members/<memberid>/corporates/<corporateid>/contests
```

But in the component explorer these routes are not displayed.
It's a bug? If not it would be very nice as a new feature!

**Comment by jtan4gov** [comment]
Any comments on this? We have not seen the routes exposed in the explorer as well. Thanks!

**Comment by tjosbon** [comment]
Same here, the nested relationships are not showing in explorer.

**Comment by jankcat** [comment]
Anyone have a workaround, maybe a way to display all active routes in a console.log?

**Comment by Lughino** [comment]
It seems that loopback is really very slow to fix problems.
I am very sorry to say because the starting point is really good, but in a world where the developments are very fast, loopback is left far behind!
Many PR to assess and make the merge, adding features slim to none, nonexistent bug fixes ..
Since it was acquired by ibm it seems that is no longer of interest to them to evolve and improve this framework.

**Comment by jtan4gov** [comment]
That's sad.

**Comment by sebastianhaas** [comment]
@Lughino  it's sad but I feel exactly the same way. I guess I have >10 discussions open with issues all around the framework, provided multiple showcase apps, even provided tests, tried to help on documentation but I don't think anybody is really taking care of this project anymore. By the end of 2016 I'd recommend anybody to move away.

**Comment by jankcat** [comment]
@sebastianhaas yup. Already moved to Parse. It has the functionality I need but most importantly it has an active community and is maintained.

**Comment by superkhau** [comment]
>It seems that loopback is really very slow to fix problems.

@sebastianhaas @Lughino @jankcat We have many active developers on the project, but unfortunately not every issue gets responded to promptly (there are many priorities to juggle -- there has been discussion in opening up more repos for the community to take over (and perhaps improve on the response times). That said, do you have any suggests on how to improve your experience?

**Comment by DAVIDFlavien** [comment]
See PR#208

**Comment by sebastianhaas** [comment]
Nest remoting still doesnt work as I would expect it.

**Comment by sebastianfelipe** [comment]
Nest remoting doesn't work. It defined the endpoints, but they don't appear on the explorer

**Comment by raymondfeng** [comment]
I'm seeing the same issue.

**Comment by raymondfeng** [comment]
I created a fix at https://github.com/strongloop/loopback-component-explorer/pull/222

**Comment by raymondfeng** [comment]
Fixed by loopback-component-explorer@5.1.0 or loopback-component-explorer@4.3.0.
